stackset.tf

```diff
@@ -0,0 +1,56 @@
+data "aws_iam_policy_document" "AWSCloudFormationStackSetAdministrationRole_assume_role_policy" {
+  count = var.enable_organization_trail ? 1 : 0
+
+  statement {
+    actions = ["sts:AssumeRole"]
+    effect  = "Allow"
+
+    principals {
+      identifiers = ["cloudformation.amazonaws.com"]
+      type        = "Service"
+    }
+  }
+}
+
+resource "aws_iam_role" "AWSCloudFormationStackSetAdministrationRole" {
+  count = var.enable_organization_trail ? 1 : 0
+
+  assume_role_policy = data.aws_iam_policy_document.AWSCloudFormationStackSetAdministrationRole_assume_role_policy[0].json
+  name               = "AWSCloudFormationStackSetAdministrationRole"
+
+  tags = local.tags
+}
+
+resource "aws_cloudformation_stack_set" "permeate_account_policy" {
+  count = var.enable_organization_trail ? 1 : 0
+
+  name             = "PermeateAccountPolicy"
+  description      = "Creates policies in all accounts of the organinzation for Expel to get basic read permissions of resources in order to better support"
```

Comment:
It feels like this is missing a word or two on the end -- "to better support" what? Investigating alerts?

Reply:
Ah yes, I got distracted as I was writing this one 😄